Documents/UserManual/userManualFrench.md

```diff
@@ -0,0 +1,265 @@
+<p align="center">
+    <img src="https://assets-global.website-files.com/656097f111c9c42578f4fa01/65648d5d98f6814022b9474a_6e5ca96fcf317623caec20ca96892327.png">
+</p>
+
+# <div style="text-align: right; text-align: down"> Mode d'emploi </div>
+# <div style="text-align: right; text-align: down"> Version 1.1 </div>
+
+<div style="page-break-after: always;"></div>
+
+<details open>
+<summary>Sommaire</summary>
+
+- [ Mode d'emploi ](#-mode-demploi-)
+- [ Version 1.1 ](#-version-11-)
+- [Mesures de sécurité](#mesures-de-sécurité)
+  - [Avertissements](#avertissements)
+- [Premières Etapes](#premières-etapes)
+  - [Contenu de la boîte](#contenu-de-la-boîte)
+  - [Chargez votre SportShield](#chargez-votre-sportshield)
+  - [Set up SportShield](#set-up-sportshield)
+- [Comment verrouiller SportShield](#comment-verrouiller-sportshield)
+- [Comment déverrouiller SportShield](#comment-déverrouiller-sportshield)
+- [Alarme](#alarme)
+- [App](#app)
+- [Notifications](#notifications)
+- [General Infos and Spécifications techniques](#general-infos-and-spécifications-techniques)
+  - [Composants](#composants)
+  - [Garantie de robustesse](#garantie-de-robustesse)
+    - [Antivol Câble](#antivol-câble)
```

Comment:
It seems to me that you're using the English structure for a French sentence, which leads to a wall.
Replace "Antivol Câble" with "Câble d'Antivol" (sound French.)